clients/da-vinci-client/src/main/java/com/linkedin/davinci/kafka/consumer/LeaderFollowerStoreIngestionTask.java

```diff
@@ -1524,11 +1524,9 @@ protected static void checkAndHandleUpstreamOffsetRewind(
        * otherwise, don't fail the push job, it's streaming ingestion now so it's serving online traffic already.
        */
       String logMsg = String.format(
-          "{} partition {} received message at offset {} with upstreamOffset: {};"
```

Comment:
Can we use String concat instead of format here?

Reply:
Also String concat internally transforms to StringBuilder which performs 3x better than `format`

Reply:
This is not in the hot path though. It has lots of variables, concat would look like a mess :D

Reply:
But would be safer :)